omero_search_engine/validation/psql_templates.py

```diff
@@ -41,6 +41,39 @@ def substitute(self, **kwargs):
         return super(SqlSearchEngineTemplate, self).substitute(kwargs)
 
 
+# get aviable values for an image key
+query_images_aviable_values_for_key = Template(
```

Comment:
typo in the variable name
it should be ``query_images_available_values_for_key``